src/app/login/login.component.css

```diff
@@ -0,0 +1,40 @@
+
+.container{
+  text-align:center;
+}
+
+h1{
+  margin-bottom: 5%;
+}
+
+form{
+  text-align: left;
+  margin-left:25%;
+  margin-right:25%;
+}
+
+label{
+  font-size: large;
+  margin-bottom:-8px;
+}
+
+/* CSS style pulled from:
+https://www.w3schools.com/howto/howto_css_login_form.asp
+*/
+input[type=text], input[type=password] {
+  width: 100%;
+  padding: 12px 20px;
+  margin: 8px auto;
+  display: inline-block;
+  border: 1px solid #ccc;
+  box-sizing: border-box;
+}
+
+button{
```

Comment:
This is applying to all buttons on the login page, which is fine for now but will need to be changed if we want a different style of button. In the future, using class based identifies would be better

Reply:
Yeah, I copy the CSS from the registration CSS to ensure both pages are similar. We could merge two files into one in the future.